Library/Homebrew/build_environment.rb

```diff
@@ -46,7 +46,13 @@ def build_env_keys(env)
       HOMEBREW_SDKROOT HOMEBREW_BUILD_FROM_SOURCE
       MAKE GIT CPP
       ACLOCAL_PATH PATH CPATH
-    ].select { |key| env.key?(key) }
+    ]
+  end
+
+  def os_specific_build_env_keys; end
```

Comment:
This should return an empty array or the code below won't work in the generic case.

Reply:
`brew tests --generic` reveals this error. It's probably worth always running locally for cross-platform changes.